### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by mychaelgo** [mychaelgo/gojek#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ksmora** [comment]
hello sir I want to make an application that compares prices from gojek and grab, so this application will help users choose the best service in terms of price
1. can it be made using this API
2. can you help me

**Reply by ksmora** [reply]
I forgot to introduce myself
I am a student in Bandung who is in the process of completing a thesis
